### PR TITLE
Feature/publish docker on tag

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description

Build a docker image for phoenix using build artifacts from drone and push the image to docker hub

## Motivation and Context
Provide an image for phoenix

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -4963,6 +4963,7 @@ steps:
   image: toolhippie/docker:latest
   commands:
   - docker build -t owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} .
+  - docker push owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER}
```
Comment (tboerger) [inline .drone.yml]: It should not be published with this tag, please define a proper tag, `:latest` could be built from master branch, a git tag should be autoi tagged, e.g. v1.1.0 gets tagged as `:1`, `:1.1` and `:1.1.0`

File: Makefile.release [hunk 2]
```diff
@@ -35,7 +35,7 @@ help:
 
 .PHONY: dist
 dist:                       ## Build distribution
-dist: distdir javascript oc-app distconf sign package
+dist: distdir javascript oc-app distconf sign package docker-image
```
Comment (tboerger) [inline Makefile.release]: Why should that be part of the release makefile?

File: .drone.starlark [hunk 3]
```diff
@@ -566,7 +566,8 @@ def buildDockerImage():
 			'DOCKER_HOST': 'tcp://docker:2375'
 		},
 		'commands': [
-			'docker build -t owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} .'
+			'docker build -t owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} .',
+			'docker push owncloud/phoenix:latest'
```
Comment (tboerger) [inline .drone.starlark]: You need to add `docker tag owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} owncloud/phoenix:latest`

File: Makefile.release (hunk 2 above)
Comment (DeepDiver1975) [reply Makefile.release]: I thought when doing a release having a docker image being spit out is reasonable ....
... this all needs more automation and then the release makefile can die ...

File: .drone.yml (hunk 1 above)
Comment (DeepDiver1975) [reply .drone.yml]: having proper automated releases on tag is what we need to do . yes

- pushing a tag
- build docker image and public
- build appstore tarzg and publish as github release (marketplace upload can happen manually)

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6532/

<p>





</p></details>

File: .drone.starlark (hunk 3 above)
Comment (DeepDiver1975) [reply .drone.starlark]: done

**Pull request merged**